source/server-selection/server-selection.rst

```diff
@@ -1041,12 +1043,28 @@ The shortest average round trip time (RTT) from among suitable servers anchors
 one end of the latency window (``A``).  The other end is determined by adding
 ``localThresholdMS`` (``B = A + localThresholdMS``).
 
-A server MUST be selected randomly from among suitable servers that have an
-average RTT (``RTT``) within the latency window (i.e. ``A ≤ RTT ≤ B``).
+A server MUST be selected from among suitable servers that have an average RTT
+(``RTT``) within the latency window (i.e. ``A ≤ RTT ≤ B``). In other words, the
+suitable server with the shortest average RTT is **always** a possible choice.
+Other servers could be chosen if their average RTTs are no more than
+``localThresholdMS`` more than the shortest average RTT.
+
+A server MUST be selected according to the following algorithm, which is a
+slight modification of the `"Power of Two Random Choices with Min Connections"
+<https://www.nginx.com/blog/nginx-power-of-two-choices-load-balancing-algorithm/>`_
+load balancing algorithm. The steps are as follows:
+
+1. Select two servers at random from the set of available servers in the latency
+   window
+
+2. If ``abs(server1.pool.activeConnectionCount - server2.pool.activeConnectionCount) > 0.05*maxPoolSize``,
```

Comment:
Should `0.05*maxPoolSize` be updated to `max(0.05*maxPoolSize, 1)`?

Reply:
fixed